web-admin/src/features/billing/banner/BillingBannerManagerForAdmins.svelte

```diff
@@ -33,9 +36,13 @@
     });
   }
 
-  $: if ($billingIssueMessage.data) {
+  $: showBillingIssueBanner($billingIssueMessage.data);
+  onMount(() => {
+    // There is a race condition where BannerCenter is mounted after the above statement is run.
+    // So call showBillingIssueBanner again to make sure banner is shown.
+    // TODO: we should probably save the last event args and re-fire them when a listener added
     showBillingIssueBanner($billingIssueMessage.data);
-  }
+  });
```

Comment:
`onMount` is run _before_ any reactive statements, so the comment seems incorrect?

Reply:
Where are you getting that? Here is a repl that goes against it: https://svelte.dev/playground/51e65be2ee464ee1a9ec6c0b1d0e2bd3?version=5.20.4

Reply:
Oh my mistake: the reactive statements are run _after other code in the <script>_, but I didn't realize they are also run _before the component markup is rendered_

Docs:
<img width="848" alt="image" src="https://github.com/user-attachments/assets/11e0e51e-c2f5-4d6b-8d29-7d5136f17d10" />